examples/parser-basic/README.md

```diff
@@ -0,0 +1,12 @@
+# Parser Example
+
+A custom [parser](https://www.techopedia.com/definition/3854/parser) for counting letters.
+
+platform: [roc-lang/basic-cli 0.2.1](https://github.com/roc-lang/basic-cli/tree/0.2.1)
```

Comment:
I feel like these look a bit bare on the page. I like the idea of having them split out, but I might play with this to see if there is a nicer way to present them.